bool is_multiple
uint8 frequency
uint8 type
